src/types.ts

```diff
@@ -409,49 +409,50 @@ export type InternalNestedTransactionFunction = <T>(
   transactionRetryLimit?: number,
 ) => Promise<T>;
 
-// eslint-disable-next-line @typescript-eslint/no-empty-interface, @typescript-eslint/consistent-type-definitions, @typescript-eslint/no-unused-vars
-export interface TaggedTemplateLiteralInvocation<Result extends UserQueryResultRow = QueryResultRow> extends SqlSqlToken { }
+type MixedRow = QueryResultRow | ZodTypeAny;
```

Comment:
Two very distinct types for such a core piece of the library, and so tied up in zod's abstractions doesn't quite feel right to me. I think either of my previous to suggestions would remove the need for this big change. I really don't think a breaking change is necessary!